help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+* Snap Inc does not support renaming segments. To rename a segment, you must deactivate, rename it, and then activate it.
+* It is not possible to define a retention period for an audience segment's members. All members have lifetime retention and will be in the segment until they are removed..
+
+
+## Supported identities {#supported-identities}
+
+The *Snap Inc* destination supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+All identifiers sent to the *Snap Inc* destination must be hashed in SHA-256 format. To hash plain text identifiers before sending them to the destination, check the [!UICONTROL Apply transformation] option when mapping target identifiers for the destination. 
+
+**Unhashed identifiers will not be accepted by the Snap Inc destination and sending them could cause errors.**
+
+>[!IMPORTANT]
+> 
+> The Snap Inc destination does not support multiple identities. Please select only one identity.
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+|Email Address |SHA-256 hashed email address | Map email addresses into the target identity field *emailAddress*.
+|Phone Number |SHA-256 hashed phone number| Map email addresses into the target identity field *phoneNumber*.|
+|GAID |SHA-256 hashed Google Advertising ID| Map Google Advertising IDs into the target identity field *gaid*.|
+|IDFA |SHA-256 hashed Apple Advertising ID| Map Apple Advertising IDs into the target identity field *idfa*.
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the *YOURDESTINATION* destination.|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connecting to Snap Inc {#connect}
+
+>[!IMPORTANT]
+> 
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, follow these steps:
+
+1. Find the *Snap Inc* destination from the Adobe Experience Platform's Destination Catalog and click on **Set Up**.
```

Comment:
_really_ minor: we tend to avoid the term click, considering that not all users might be using a mouse. 

Suggested rewording: 
Find the *Snap Inc* destination from the Adobe Experience Platform destination catalog and select **Set Up**.